src/ResourceManager/Resources/Commands.Resources/Models.Authorization/PSADCredentialWrapper.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.Azure.Graph.RBAC.Version1_6.ActiveDirectory;
+using System.Security;
+
+namespace Microsoft.Azure.Commands.Resources.Models.Authorization
+{
+    public class PSADCredentialWrapper : PSADCredential
+    {
+        public PSADCredentialWrapper(PSADCredential cred)
```

Comment:
@maddieclayton I would add a null-check at the beginning of this constructor to be safe